libdnf/base/goal.cpp

```diff
@@ -510,17 +522,40 @@ GoalProblem Goal::Impl::report_not_found(
         pkg_spec, false, settings.with_nevra, settings.with_provides, settings.with_filenames, true, settings.forms);
     if (!nevra_pair_reports.first) {
         // RPM was not excluded or there is no related srpm
-        add_rpm_goal_report(action, libdnf::GoalProblem::NOT_FOUND, settings, pkg_spec, strict);
+        add_rpm_goal_report(action, libdnf::GoalProblem::NOT_FOUND, settings, pkg_spec, {}, strict);
```

Comment:
The `libdnf::rpm` namespace is still repeated. But the code is already part of the `libdnf` namespace. So `rpm` is enough.